packages/server/assets/core/typedefs/modelsAndVersions.graphql

```diff
@@ -0,0 +1,180 @@
+extend type Project {
+  model(id: String!): Model # note we're not using branch (model) names anymore
+  models(limit: Int! = 20, cursor: String, query: ModelQuery): ModelCollection
```

Comment:
model collection default sort by editedAt
model editedAt should be updated on ~~commit~~ version creation